supply/lib/supply/options.rb

```diff
@@ -147,35 +174,41 @@ def self.available_options
                                      optional: true,
                                      description: "Whether to skip uploading APK",
                                      is_string: false,
-                                     default_value: false),
+                                     default_value: true),
         FastlaneCore::ConfigItem.new(key: :skip_upload_aab,
                                      env_name: "SUPPLY_SKIP_UPLOAD_AAB",
                                      optional: true,
                                      description: "Whether to skip uploading AAB",
                                      is_string: false,
-                                     default_value: false),
+                                     default_value: true),
         FastlaneCore::ConfigItem.new(key: :skip_upload_metadata,
                                      env_name: "SUPPLY_SKIP_UPLOAD_METADATA",
                                      optional: true,
-                                     description: "Whether to skip uploading metadata",
+                                     description: "Whether to skip uploading metadata, changelogs not included",
                                      is_string: false,
-                                     default_value: false),
+                                    default_value: true),
+        FastlaneCore::ConfigItem.new(key: :skip_upload_changelogs,
+                                     env_name: "SUPPLY_SKIP_UPLOAD_CHANGELOGS",
+                                     optional: true,
+                                     description: "Whether to skip uploading changelogs",
+                                     is_string: false,
+                                     default_value: true),
```

Comment:
Maybe also make false? But maybe also just get rid of and use `skip_upload_metadata `